src/components/HookCounter/HookCounter.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react'
+
+function HookCounter() {
+  const [count, setCount] = useState(0)
```

Comment:
when using useState, try to define a type for state variables as well `useState<number>(0)` something like this